include/irr/asset/ICPUMeshBuffer.h

```diff
@@ -84,11 +84,37 @@ class ICPUMeshBuffer : public IMeshBuffer<ICPUBuffer, ICPUDescriptorSet, ICPURen
     {
         return &m_indexBufferBinding;
     }	
+	[[nodiscard]] inline bool setVertexBufferBinding(SBufferBinding&& bufferBinding, uint32_t bindingIndex, uint32_t stride, E_VERTEX_INPUT_RATE inputRate = E_VERTEX_INPUT_RATE::EVIR_PER_VERTEX)
```

Comment:
whats the [[nodiscard]] for?

Plus we use C++14 in the library

Reply:
To ensure that user will check if operation is valid 
Okay, I'll remove it